build.sbt

```diff
@@ -687,6 +691,8 @@ lazy val `akka-discovery-service-locator-scaladsl` = (project in file("akka-serv
   .dependsOn(`akka-discovery-service-locator-core`)
   .dependsOn(`client-scaladsl`)
   .settings(runtimeLibCommon: _*)
+  // TODO: after this get released, we must enable the mima settings
+  // that artifact doesn't exist in previous jar, so it doesn't make sense to check it
```

Comment:
@renatocaval Do we have an issue for version 1.4.13 to review these `TODO`'s?

(I'm not sure I understood your train of thought so I mentioned `1.4.13` and maybe you had another version in mind when you wrote the comment)